tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -76,7 +76,7 @@ static DatadogLogging()
                        .MinimumLevel.ControlledBy(LoggingLevelSwitch)
                        .WriteTo.File(
                             managedLogPath,
-                            outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] {Message:lj}{Exception}{Properties}{NewLine}",
+                            outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] {Message:lj}{Exception} {Properties}{NewLine}",
```

Comment:
In that case shouldn't we add one also before the exception?

Reply:
AFAICT, the exception is formatted on a new-line anyway, so shouldn't be a problem e.g. 
![image](https://user-images.githubusercontent.com/18755388/143604830-8d4cd19f-e13e-46a5-a534-436218ad147f.png)